snappl/psf.py

```diff
@@ -337,6 +341,29 @@ def write( self, filepath ):
         # TODO : check overwriting etc.
         yaml.dump( out, open( filepath, 'w' ) )
 
+class A25ePSF( YamlSerialized_OversampledImagePSF ):
+
+    def __init__( self, band, sca, x, y, *args, **kwargs ):
+
+        super().__init__( *args, **kwargs )
+        
+        cfg = Config.get()
+        basepath = pathlib.Path( cfg.value( 'photometry.snappl.A25ePSF_path' ) )
+
+        grid_centers = np.linspace(0.5 * 511, 4088 - 0.5 * 511, 8)
+
+        dist_x = np.abs(grid_centers - x)
+        dist_y = np.abs(grid_centers - y)
+
+        x_idx = np.argmin(dist_x)
+        y_idx = np.argmin(dist_y)
+
+        x_cen = grid_centers[x_idx]
+        y_cen = grid_centers[y_idx]
+        
+        psfpath = basepath / band / str(sca) / f'511_{x_cen:.1f}_{y_cen:.1f}_-_19.0_21.5_-_{band}_{sca}.psf'
```

Comment:
How do we know about 19.0 and 21.5?  Are those fixed.  Should those be variables.
Use the variable to define the `511` part of the string